setup.cfg

```diff
@@ -0,0 +1,46 @@
+[metadata]
+name = qutip-qip
+url = https://github.com/qutip/qutip-qip
+description = The QuTiP quantum information processing package
+long_description = file: README.md
+long_description_content_type = text/markdown; variant=GFM
+keywords = quantum, physics, dynamics
+license = BSD 3-Clause License
+license_files = LICENSE
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: Microsoft :: Windows
+author = Alexander Pitchford, Paul D. Nation, Robert J. Johansson, Chris Granade, Arne Grimsmo, Nathan Shammah, Shahnawaz Ahmed, Neill Lambert, Eric Giguere, Boxi Li, Jake Lishman
+author_email = qutip-admin@googlegroups.com
+platforms = Linux, Mac OSX, Unix, Windows
+version = 0.1.0
+
+[options]
+package_dir=
+    =src
+packages = find:
+zip_safe = False
+include_package_data = True
+install_requires =
+    numpy>=1.16.6
+    scipy>=1.0
```

Comment:
You're probably don't actually have any version constraints - QuTiP currently requires `numpy>=1.16.6` because that had some changes to the C-API.  That said, v1.16 of Numpy and v1.0 of Scipy are both quite old now, so this isn't much more than no constraint at all.